not-to-release/sources/reviews/203196.xml.conllu

```diff
@@ -8,7 +8,7 @@
 # newpar id = reviews-203196-p0002
 # text = VISAKHA INDUSTRIAL GASES PVT. LTD., location at google maps.
 1	VISAKHA	VISAKHA	PROPN	NNP	Number=Sing	5	compound	5:compound	_
-2	INDUSTRIAL	INDUSTRIAL	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+2	INDUSTRIAL	INDUSTRIAL	ADJ	NNP	Degree=Pos	5	amod	5:amod	_
 3	GASES	GASES	PROPN	NNPS	Number=Plur	5	compound	5:compound	_
 4	PVT.	PVT.	PROPN	NNP	Number=Sing	5	compound	5:compound	_
 5	LTD.	LTD.	PROPN	NNP	Number=Sing	0	root	0:root	SpaceAfter=No
```

Comment:
In the same bucket as Roman numerals as heads, LTD. as head looks off to me

Reply:
#167